iree/turbine/kernel/lang/global_symbols.py

```diff
@@ -10,6 +10,8 @@
 WORKGROUP_0 = index_symbol("$WG0")
 WORKGROUP_1 = index_symbol("$WG1")
 WORKGROUP_2 = index_symbol("$WG2")
+WORKGROUP_3 = index_symbol("$WG3")
+WORKGROUP_4 = index_symbol("$WG4")
```

Comment:
Doesn't the GPU typically only have 3 dims? 😆

Reply:
They do :) This is just to keep it easier for the kernel author. All workgroup dimensions >= 2 are obtained by delinearizing workgroup id z.